documentation/installation.md

```diff
@@ -37,10 +37,12 @@ Instead of installing whole repository with all dependencies, it's recommended t
 | [flow-php/core](/documentation/components/core/core.md)                                                        | [![Total Downloads](https://poser.pugx.org/flow-php/etl/downloads)](https://packagist.org/packages/flow-php/etl)                                                       | [![Latest Stable Version](https://poser.pugx.org/flow-php/etl/v/stable)](https://packagist.org/packages/flow-php/etl)                                                       | [![](https://flow-php.github.io/commits-since-release-counter/flow-php/etl.svg)](https://github.com/flow-php/etl/tags)                                                       |
 | [flow-php/cli](/documentation/components/cli/docs.md)                                                          | [![Total Downloads](https://poser.pugx.org/flow-php/cli/downloads)](https://packagist.org/packages/flow-php/cli)                                                       | [![Latest Stable Version](https://poser.pugx.org/flow-php/cli/v/stable)](https://packagist.org/packages/flow-php/cli)                                                       | [![](https://flow-php.github.io/commits-since-release-counter/flow-php/cli.svg)](https://github.com/flow-php/cli/tags)                                                       |
 | **Adapters**                                                                                                   |                                                                                                                                                                        |                                                                                                                                                                             |                                                                                                                                                                              |
+| [flow-php/etl-adapter-avro](/documentation/components/adapters/avro.md)                                        | [![Total Downloads](https://poser.pugx.org/flow-php/etl-adapter-avro/downloads)](https://packagist.org/packages/flow-php/etl-adapter-avro)                             | [![Latest Stable Version](https://poser.pugx.org/flow-php/etl-adapter-avro/v/stable)](https://packagist.org/packages/flow-php/etl-adapter-avro)                             | [![](https://flow-php.github.io/commits-since-release-counter/flow-php/etl-adapter-avro.svg)](https://github.com/flow-php/etl-adapter-avro/tags)                             |
```

Comment:
could you actually remove Avro? It's currently not maintained since the php avro library wasn't working that well. I'm going to rewrite it in the future like I did with Parquet, but for now I think we should not expose Avro anywhere